pkg/apis/scaler/v1alpha1/types_demand.go

```diff
@@ -47,7 +49,8 @@ type DemandSpec struct {
 
 // DemandStatus represents the status a demand object is in
 type DemandStatus struct {
-	Phase string `json:"phase"`
+	Phase              string    `json:"phase"`
+	LastTransitionTime time.Time `json:"last-transition-time"`
```

Comment:
we should use metav1.Time which is a k8s wrapper around the golang time.

Reply:
done